src/bp/core/config/config-loader.ts

```diff
@@ -70,6 +89,25 @@ export class ConfigProvider {
     return config
   }
 
+  private _makeBPConfigEnvKey(option: string): string {
+    return `BP_CONFIG_${option}`.toUpperCase()
+  }
+
+  private async _loadBotpressConfigFromEnv(currentConfig: BotpressConfig): Promise<PartialDeep<BotpressConfig>> {
+    const configOverrides: PartialDeep<BotpressConfig> = {}
+    const weakSchema = await this._getBotpressConfigSchema()
+    const options = await getValidJsonSchemaProperties(weakSchema as SchemaNode, currentConfig)
+    for (const option of options) {
+      const envKey = this._makeBPConfigEnvKey(option)
+      const value = getValueFromEnvKey(envKey)
+      if (value) {
```

Comment:
Shouldn't this be `value !== undefined` ? Otherwise you can't set a value to "false"